src/test/java/com/squareup/javapoet/MethodSpecTest.java

```diff
@@ -456,6 +456,44 @@ static void staticMethod() {
         "}\n");
   }
 
+  @Test public void multilineComments() {
+    MethodSpec main = MethodSpec.methodBuilder("main")
+        .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
+        .returns(void.class)
+        .addParameter(String[].class, "args")
+        .addComment("Hello\nmultiline\ncomments!")
+        .addStatement("$T.out.println($S)", System.class, "Hello, JavaPoet!")
+        .build();
+
+    assertThat(main.toString()).isEqualTo("" +
+        "public static void main(java.lang.String[] args) {\n" +
+        "  /*\n" +
```

Comment:
I suggest using the following syntax for inline comments:
```java
public static void main(String[] args) {
  // Hello
  // multiline
  // comments!
}
```
The `/** */` is most frequently used for Javadoc, so generating it inside a method body feels awkward.

Reply:
While `/**` is for javadoc, `/*` is actually the intended multi-line comment syntax: https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141999.html

Reply:
> The last revision to this document was made on April 20, 1999

Reply:
It's fine if you would prefer not to have it, but that syntax is valid multi-line comment syntax. You can look at other sources and they all distinguish multi-line comments and JavaDoc comments by the extra asterisk. It's not a valid argument to dismiss this by claiming it as JavaDoc.

Reply:
The multiline comment syntax is valid Java as @octylFractal pointed out. It's common to see single line comments completely replaced by block comments throughout source code. The one that's used for JavaDocs begin with slash followed by two asterisks. /* */ and  /** */ are two entirely different things. See [here](https://stackoverflow.com/questions/29815636/and-in-java-comments/29815698) for more information.

Reply:
Yeah, I totally confused the two. My preference is still for `//` - that's less lines and feels more common to me. I haven't seen the `/* */` form used in method bodies in a long while.